python/lsst/ip/diffim/getTemplate.py

```diff
@@ -222,11 +222,11 @@ def run(self, exposure, sensorRef, templateIdList):
         ----------
         exposure :  `lsst.afw.image.Exposure`
             exposure (unused)
-        sensorRef : TYPE
-            a Butler data reference
-        templateIdList : TYPE
-            list of data ids, which should contain a single item.
-            If there are multiple items, only the first is used.
+        sensorRef : `lsst.daf.persistence.ButlerDataRef`
+            Data reference of the calexp(s) to subtract from.
+        templateIdList : `lsst.daf.persistence.ButlerDataRef`
```

Comment:
`list` of `lsst.daf.persistence.ButlerDataRef`
And you should include the deleted comment that: "If there are multiple items, only the first is used."